src/pages/superadmin/tableComponent/__tests__/TableComponent.spec.tsx

```diff
@@ -159,9 +159,8 @@ it('renders each element in the table in the document', () => {
     expect(getByText(bounty.title)).toBeInTheDocument();
     expect(getByText(dates[index])).toBeInTheDocument();
     // expect(getByText(String(bounty.dtgp))).toBeInTheDocument();
-    expect(getByText(bounty.assignee)).toBeInTheDocument();
+    expect(getByText(bounty.assignee_alias)).toBeInTheDocument();
     // expect(getByText(bounty.provider)).toBeInTheDocument();
-    expect(getByText(bounty.organization)).toBeInTheDocument();
```

Comment:
we should replace this test with the org name instead of deleting it

Reply:
Yes, but interface don't contain organisation_name adding it will break other components so Changed type to any of mockbounties will push it rn